tensorflow/python/keras/applications/mobilenet_v2.py

```diff
@@ -157,6 +157,17 @@ def MobileNetV2(input_shape=None,
     ValueError: in case of invalid argument for `weights`,
       or invalid input shape or invalid alpha, rows when
       weights='imagenet'
+      
+  Usage Example:
+  >>> from tensorflow.keras.applications import MobileNetV2
+  >>> IMG_SHAPE = (224, 224, 3)
+  >>> model = MobileNetV2(input_shape =IMG_SHAPE , include_top = True)
+  >>> model
+  <tensorflow.python.keras.engine.training.Model at 0x7f19e6f42d68>
```

Comment:
`0x7f19e6f42d68`won't be the same on other runs. Did you test the doctest locally?

Reply:
I tested it on colab.